ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -15,6 +15,12 @@ import { ERROR_WINDOW_CLOSED, ERROR_MISSING_PARAMS, ERROR_JWT_LOGIN } from 'vaul
 
 const component = create(form);
 const windows = [];
+const buildMessage = opts => ({
+  isTrusted: true,
+  origin: 'http://localhost:8200',
```

Comment:
I guess we'll see, but I'd be surprised if this works on CircleCI, or maybe I wouldn't be because Circle CI is a bit of a black box.

Reply:
it works! The window and events are all mocked, but the logic in the component would fail if origin from the event doesn't match which is why I added this. It could be anything, I'll change it to something generic so that is clear

Reply:
Rad.  I learned something new today. Thanks!